src/components/singleBolgPage/index.css

```diff
@@ -0,0 +1,129 @@
+.articleHeader,
+.authorImage,
+.postInfo {
+  display: flex;
```

Comment:
so usually instead of grouping them at the top like this i would put display: flex individually in each of the rules, for example line 15 i would add a display: flex and delete this one, line 22 i would add a display: flex and delete this one, and so on

because logically when someone wants to fix something in the articleHeader they don't want to look for rules in 3 diff places